internal/ethereum/event_enricher.go

```diff
@@ -47,18 +47,24 @@ func (ee *eventEnricher) filterEnrichEthLog(ctx context.Context, f *eventFilter,
 	addrMatches := f.Address == nil || bytes.Equal(ethLog.Address[:], f.Address[:])
 	if !topicMatches || !addrMatches {
 		log.L(ctx).Debugf("skipping event '%s' topicMatches=%t addrMatches=%t", protoID, topicMatches, addrMatches)
-		return nil, matched, decoded, nil
```

Comment:
Do `bool`s default initialise to `false` and this is just to tidy up the code, not change the logic?